src/core/qgsquick/qgsquickmapcanvasmap.h

```diff
@@ -115,6 +115,11 @@ class QgsQuickMapCanvasMap : public QQuickItem
     //! \copydoc QgsQuickMapCanvasMap::incrementalRendering
     void setIncrementalRendering( bool incrementalRendering );
 
+    /**
+     * Returns an image of the last successful map canvas rendering
+     */
+    const QImage image() const { return mImage; }
```

Comment:
```suggestion
    QImage image() const { return mImage; }
```

No reason to force the return value to be const